yt/data_objects/data_containers.py

```diff
@@ -1785,7 +1785,332 @@ def cut_region(self, field_cuts, field_parameters=None):
         cr = self.ds.cut_region(self, field_cuts,
                                 field_parameters=field_parameters)
         return cr
+   
+    def exclude_above(self, field, value, units=None):
+        """
+        This function will return a YTCutRegion where all of the regions
+        whose field is above a given value are masked.
+
+        Parameters
+        ----------
+        field : string
+            The field in which the conditional will be applied.
+        value : float
+            The minimum value that will not be masked in the output
+            YTCutRegion.
+        units : string or None
+            The units of the value threshold. None will use the default units
+            given in the field.
+
+        Returns
```

Comment:
Could you add an Example for each one of these functions?  This is likely going to be the place that most users come for documentation on how to use this.

Reply:
I added an example for each of the added functions in the latest commit.